importScripts/sql/transformations/ObjectsExportView.sql

```diff
@@ -60,7 +61,15 @@ CREATE OR REPLACE VIEW ObjectsExport AS
         NULLIF(TRIM(o.InsuranceValue), '')                        AS InsuranceValue,
         NULLIF(TRIM(o.ItemDates), '')                             AS ItemDates,
         NULLIF(TRIM(o.ItemName), '')                              AS ItemName,
-        NULLIF(TRIM(o.ItemType), '')                              AS ItemType,
+        CASE WHEN o.ItemType = '' AND o.AccessionPrefix = 'P'
```

Comment:
Source data doesn't have ItemType filled in for some records. This works out what it should be based on the AccessionPrefix